### ISSUE TITLE
How to enter comma (function parameter separator)?

### ISSUE THREAD

**Issue by GLaDOS1105** [daniel3735928559/guppy#118]
How could I enter a comma or semicolon in Guppy?
For example,
![example comma](https://latex.codecogs.com/png.latex?%5Clarge%20f%28x%2C%20y%29%20%3D%20x%20&plus;%20y)
![example semicolon](https://latex.codecogs.com/png.latex?%5Clarge%20lik%28%5Ctheta%3B%20X%29%20%3D%20P%28X%20%7C%20%5Ctheta%29)
Perhaps, there is a way to do this in Guppy but I cannot find it.

**Comment by ibreaker14** [comment]
@daniel3735928559 is there a reason why commas are blocked? Also, is there a workaround for this?

**Comment by daniel3735928559** [comment]
Yeah--beacuse guppy has to know what everything you enter means. For example, if you type `f ( x )` guppy interprets this as f*(x). 

It doesn't have a way to tell that you mean f is a function and not a variable unless you tell it so. 

The current way of doing this (which is admittedly a little noninuitive) is to type `eval` (as in, "evaluate a function"). This will give you a function object where you can input the name of the function as well as an argument list (separated by commas). If you want some specific variable like `f` to be treated as a function, define a symbol f analogous to the `eval` symbol in symbols.json. 

Note that if you enter `eval f u n c [rightarrow] x` this will be rendered as func(x) but will be interpreted as "evaluate the function f*u*n*c on the argument x".
